fury/actor.py

```diff
@@ -2740,3 +2742,103 @@ def callback(
                     block="light")
 
     return sq_actor
+
+
+def bitmap_labels(
+        centers,
+        labels,
+        colors=(0, 1, 0),
+        scales=1,
+        align='center',
+        x_offset_ratio=1,
+        y_offset_ratio=1,
+        font_size=7,
+        font_path=None,
+        atlas_size=(1024, 1024)
+        ):
+    """Create a bitmap label actor that always faces the camera.
+
+    Parameters
+    ----------
+    centers : ndarray, shape (N, 3)
+    labels  : list
+        list of strings
+    colors : array or ndarray
+    scales : float
+    align : str, {left, right, center}
+    x_offset_ratio : float
+        Percentage of the width to offset the labels on the x axis.
+    y_offset_ratio : float
+        Percentage of the height to offset the labels on the y axis.
+    font_size : int, optional
+        size of the text
+    font_path : str, optional
+        str of path to font file
+    atlas_size : tuple, optional
+        (width, height) of the atlas image to use for the labels.
+        Default is (1024, 1024). You can change this if you change the
+        font size or font path default parameters.
+
+    Returns
+    -------
+    vtkActor
+
+    """
+    img_arr, char2pos = text_tools.create_bitmap_font(
+        font_size=font_size, font_path=font_path,
+        atlas_size=atlas_size, show=False)
+    padding, labels_positions,\
+        uv, relative_sizes = text_tools.get_positions_labels_billboards(
```

Comment:
Positions are not going to be the final positions of each  quad (no padding applied to them) vtk clipping will remove the object if the center of the text fall off the visible area. This can be fixed later via an Issue.